Genome Tracker terms

Anoosha-Sehar opened the pull request.

Anoosha-Sehar committed:
Genome Tracker terms

ddooley commented on src/ontology/imports/foodon_product_import.owl:
```diff
@@ -34131,6 +34131,81 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://www.merriam-webster.com/dictionary/caseinate</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-03-04T10:37:22Z</dc:date>
+        <rdfs:label xml:lang="en">caseinate</rdfs:label>
```
I'm wondering whether "caseinate" was requested?  I suspect a better parent is Casein, http://purl.obolibrary.org/obo/FOODON_03420180 because Wikipedia says https://en.wikipedia.org/wiki/Casein "The most common form of casein is sodium caseinate."

Anoosha-Sehar replied:
Yes, the requested term is 'caseinate'. I even saw this term in the template sheet, which was shown during Genome Tracker's last meeting. 
That's right, sodium caseinate is the most common form of casein, and I think the second common is calcium caseinate which is used to increase the calcium content in food products. 
`Casein`: A protein substance found in milk or in the seeds of leguminous plants.
`Caseinate`: Any of several salts derived from casein via the coagulation of milk protein. 
It looks to me that a caseinate is a form of casein. I think caseinate can be a sub-class of 'casein' (if we want that)? Casein has other forms as well, like casein-based glues, but several foods, creamers, and toppings all contain a variety of caseinates.

Anoosha-Sehar committed:
Updated 'casein'

Added as a sub-class of 'food product component' and a parent class of 'sodium caseinate'.
